src/pages/RepoPage/FailedTestsTab/FailedTestsPage/FailedTestsTable/FailedTestsTable.tsx

```diff
@@ -153,26 +216,84 @@ const FailedTestsTable = () => {
     ordering: getSortingOption(sorting),
     filters: {
       branch: branch ? getDecodedBranch(branch) : undefined,
+      flags: flags as string[],
+      // eslint-disable-next-line camelcase
+      test_suites: testSuites as string[],
+      parameter: queryParams?.parameter as TestResultsFilterParameterType,
+      interval: queryParams?.historicalTrend as MeasurementInterval,
+      term: queryParams?.term as string,
     },
     opts: {
       suspense: false,
     },
   })
 
+  const isDefaultBranch = testData?.defaultBranch === branch
+  const isTeamOrFreePlan =
+    isTeamPlan(testData?.plan) || isFreePlan(testData?.plan)
+  // Only show flake rate column when on default branch for pro / enterprise plans or public repos
+  const hideFlakeRate =
+    (isTeamOrFreePlan && testData?.private) || (!!branch && !isDefaultBranch)
+
   const tableData = useMemo(() => {
-    return testData?.testResults
-  }, [testData])
+    if (!testData?.testResults) return []
+
+    return (
+      testData.testResults.map((result) => {
+        const value = (result.flakeRate ?? 0) * 100
+        const isFlakeInt = Number.isInteger(value)
+
+        const FlakeRateContent = (
+          <Tooltip delayDuration={0} skipDelayDuration={100}>
+            <Tooltip.Root>
+              <Tooltip.Trigger className="underline decoration-dotted decoration-1 underline-offset-4">
+                {isFlakeInt ? `${value}%` : `${value.toFixed(2)}%`}
+              </Tooltip.Trigger>
+              <Tooltip.Portal>
+                <Tooltip.Content
+                  className="bg-ds-gray-primary p-2 text-xs text-ds-gray-octonary"
+                  side="right"
+                >
+                  {result.totalPassCount} Passed, {result.totalFailCount} Failed
+                  ({result.totalFlakyFailCount} Flaky), {result.totalSkipCount}{' '}
+                  Skipped
+                  <Tooltip.Arrow className="size-4 fill-ds-gray-primary" />
+                </Tooltip.Content>
+              </Tooltip.Portal>
+            </Tooltip.Root>
+          </Tooltip>
+        )
+
+        return {
+          name: result.name,
+          avgDuration: result.avgDuration,
+          failureRate: result.failureRate,
+          flakeRate: FlakeRateContent,
+          commitsFailed: result.commitsFailed,
+          updatedAt: result.updatedAt,
+        }
+      }) ?? []
```

Comment:
nit: Is this nullish coalescing operator needed here?

Reply:
good catch! With line 239 its not needed, we can make an update